README.md

```diff
@@ -1,9 +1,9 @@
 # Apery
 
-[![Travis][travis-shield]][travis-link]
+[![CI][action-shield]][action-link]
```

Comment:
This diff if very mysterious to me. Which change in the config files did trigger it? What are the consequences?

Reply:
This is just a "badge" update

Reply:
My question was actually about travis vs ci,  not about the badges themselves. But I got the answer below.